plugins/currency/nanopool/nanopool_

```diff
@@ -0,0 +1,290 @@
+                "scale": "no",
+                "info": "Balance in ETH"
+            }
+        ],
+        "hashrate" : [
+            {
+                "title" : "Hashrate of {0}".format(account_address),
+                "type" : "GAUGE",
+                "args" : "--base 1000 -l 0",
+                "fields" : ["hashrate"],
+                "info": "Current Calculated Hashrate in Mh/s"
+            }
+        ],
+        "avghashrate" : [
+            {
+                "title" : "Average Hashrate of {0}".format(account_address),
+                "type" : "GAUGE",
+                "args" : "--base 1000 -l 0",
+                "fields" : ["avg_hashrate"],
+                "info": "Average Hashrate of last hour in Mh/s"
+
+            }
+        ],
+        "worker" : [
+            {
+                "title" : "Worker"
+            }
+        ]
+    }
+    return graph_types
+
+
+def request_data():
+
+    url = "https://api.nanopool.org/v1/eth/user/{0}".format(account_address)
+
+    req = Request(url)
+    # this fake is necessary to get values, otherwise the request ends in a 403 error
+    req.add_header('User-Agent', 'Nozilla/5.0')
+    try:
+        txt = urlopen(req).read()
+    except Exception as err:
+        # there could be different exceptions like ValueError, URLError, HTTPError
+        print("API request error: {0}". format(err), file=sys.stderr)
+        exit(1)
+
+    return json.loads(txt.decode("utf-8"))
+
+
+def write_config_worker():
+    data = request_data()
+    worker_data = sorted(data["data"]["workers"], key=lambda element: element["id"])
+
+    print("multigraph worker_hashrate_{0}".format(account_address))
+    print("graph_title Hashrate in Mh/s per worker ({0})".format(account_address))
+    print("graph_args --base 1000 -l 0")
+    print("graph_vlabel Mh/s")
+    print("graph_category other")
+    print("graph_scale no")
+
+    for i, val in enumerate(worker_data):
+        worker_name = "_".join(val["id"].split())
+        print("worker_{0}_hashrate.label {1}".format(worker_name, str(val["id"])))
+        print("worker_{0}_hashrate.type GAUGE".format(worker_name))
+        print("worker_{0}_hashrate.info Hashrate of worker '{1}'".format(worker_name, str(val["id"])))
+        print("worker_{0}_hashrate.min 0".format(worker_name))
+        print("worker_{0}_hashrate.draw LINE1".format(worker_name))
+    
+    for val in worker_data:
+        print("")
+        worker_name = "_".join(val["id"].split())
+        print("multigraph worker_hashrate_{0}.worker_{1}".format(account_address, worker_name))
+        print("graph_title Hashrate in Mh/s of worker {0}".format(worker_name))
+        print("graph_args --base 1000 -l 0")
+        print("graph_vlabel Mh/s")
+        print("graph_category other")
+        print("graph_scale no")
+        print("whashrate.label hashrate")
+        print("whashrate.type GAUGE")
+        print("whashrate.info Hashrate of worker {0}".format(str(val["id"])))
```

Comment:
FYI: the parameters of the `format` function are always implicitly converted to their target data type (here: string). Thus you can remove `str`.